### ISSUE TITLE
Screen reader says the wrong count in Preferences modal due to missing `radiogroup`

### ISSUE THREAD

**Issue by lindapaiste** [processing/p5.js-web-editor#2648]
### Increasing Access

We want the editor to be fully navigable using screen readers.

### Feature enhancement details

I [downloaded a screenreader](https://www.nvaccess.org/download/) and I'm playing around and finding issues.  In the preferences modal it announces like this:

> "Autosave on checked radio button 4 of 11"

It should be announcing "1 of 2" instead of "4 of 11" (where 1 is "on" and 2 is "off").  But it uses the total count of the radio buttons in the form because we do not have the `radiogroup` attribute around each "on/off" section.

Might make sense to address this after #2323 so that we only need to change it in one (or two) places instead of many.

Resources:
https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio/
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radiogroup_role

**Comment by KrishavRajSingh** [comment]
is this open?

**Comment by Jatin24062005** [comment]
@lindapaiste I looked into this issue on both Windows and macOS using the NVDA screen reader (as mentioned in the original report), and I couldn't reproduce the problem. The screen reader correctly announces "1 of 2" and "2 of 2" for the on/off radio buttons in the preferences modal.

For further confirmation, you can also check out the testing video I recorded.
@raclim — I believe this issue has been resolved and it's safe to close it now. 

https://github.com/user-attachments/assets/7634a1f8-661a-4673-9f58-ef61de6347bf

**Comment by Nishthajain7** [comment]
@raclim This issue can be closed
